tests/dags/common/loader/test_smithsonian_unit_codes.py

```diff
@@ -52,10 +52,14 @@ def test_alert_new_unit_codes():
 def test_alert_unit_codes_from_api(postgres_with_test_unit_code_table):
     postgres_conn_id = POSTGRES_CONN_ID
     unit_code_table = SI_UNIT_CODE_TABLE
+    response_mock = Mock()
+    response_mock.json.return_value = {}
 
     with patch.object(
         si, "get_new_and_outdated_unit_codes", return_value=({"d"}, {"e"})
-    ) as mock_get_unit_codes:
+    ) as mock_get_unit_codes, patch.object(
+        si.requests, "get", return_value=response_mock
+    ):
```

Comment:
FWIW this code will be deleted in #452, but I am mocking it in the [new tests](https://github.com/WordPress/openverse-catalog/pull/452/files#diff-b25f876380bf67a37d45141dba9bf98b9e88ba7194b7e2e2e02d5678172b4840R73) so it should still be good 😄

Reply:
NICE :fire: